README-ruRU.md

```diff
@@ -269,30 +268,29 @@
   ```
 
 * <a name="plain-text-rendering"></a>
-  Prefer `render plain:` over `render text:`.
-  <sup>[[link](#plain-text-rendering)]</sup>
+  Используйте `render plain:` вместо `render text:`.
+  <sup>[[ссылка](#plain-text-rendering)]</sup>
 
   ```Ruby
-  # bad - sets MIME type to `text/html`
+  # плохо - задается MIME-тип `text/html`
   ...
```

Comment:
Please use parens for comments as used throughout this guide: "Statement (some comments)". Consistency is very important.